src/paddle.cpp

```diff
@@ -2,8 +2,27 @@
 
 Paddle::Paddle() : Entity() {}
 
+Paddle::Paddle(PaddleControls pc) {
+	controls = pc;
+}
+
 Paddle::~Paddle() {}
 
 void Paddle::update(float frameTime) {
 	Entity::update(frameTime);
+
+	float yVelocity = 0.0f;
+
+	if (input->isKeyDown(controls.up)) {
+		yVelocity -= paddleNS::VELOCITY;
+	}
+
+	if (input->isKeyDown(controls.down)) {
+		yVelocity = paddleNS::VELOCITY;
+	}
```

Comment:
I'm assuming the first option. Because the 2nd one would mean the paddle has acceleration, which probably shouldnt be the case